kv/iter.go

```diff
@@ -65,7 +65,7 @@ func DecodeValue(data []byte) ([]interface{}, error) {
 }
 
 // NextUntil applies FnKeyCmp to each entry of the iterator until meets some condition.
-// It will stop when fn returns true, or iterator is invalid or occur error
+// It will stop when fn returns true, or iterator is invalid or occur error.
```

Comment:
"error" is the subject and "occur" is the predicate. 
occur is an intransitive verb.